mne/viz/epochs.py

```diff
@@ -1006,12 +1006,8 @@ def _prepare_mne_browse_epochs(params, projs, n_channels, n_epochs, scalings,
         if title is None or len(title) == 0:
             title = ''
     fig = figure_nobar(facecolor='w', figsize=size, dpi=80)
-    fig.canvas.set_window_title('mne_browse_epochs')
+    fig.canvas.set_window_title(title if title else "Epochs")
```

Comment:
FYI can be shorter as `title or `Epochs"` but no need to change

Reply:
Oh yeah, I keep forgetting about this nice shortcut. Let's keep it as is because that's also how it is implemented in `raw.plot`.

Reply:
OK, I changed both 👍.